mesonbuild/interpreter.py

```diff
@@ -3823,8 +3823,8 @@ def add_test(self, node, args, kwargs, is_base_test):
         if not isinstance(timeout, int):
             raise InterpreterException('Timeout must be an integer.')
         protocol = kwargs.get('protocol', 'exitcode')
-        if protocol not in ('exitcode', 'tap'):
-            raise InterpreterException('Protocol must be "exitcode" or "tap".')
+        if protocol not in {'exitcode', 'tap', 'gtest'}:
+            raise InterpreterException('Protocol must be "exitcode", "tap", or "gtest".')
```

Comment:
Does this need a FeatureNew call when protocol  == gtest?

Reply:
Yup, I definitely missed that.